### PR TITLE
[MIG] partner_identification: Migration to 15.0

### PR DESCRIPTION
**Pull request by ghost**
Since the PR #1270 seems to be (almost) abandoned, I backported the work done by @marielejeune for v16 in the PR #1397

### CODE REVIEW

**Comment by rousseldenis** [comment]
@jdidderen-noviat Thanks for this.

Could you follow migration guide ? https://github.com/OCA/maintainer-tools/wiki/Migration-to-version-15.0

Especially the Full process part (inversing branches in git format-patch command) ? 

You must keep commit authorship from @marielejeune ones

**Review by njeudy — APPROVED** [review]
code and test

**Comment by njeudy** [comment]
I think @marielejeune do the work in V16 and here we patch from v14 .. so .. he add marielejeune as contributor .. that ok for me I think :)

**Comment by ghost** [comment]
@rousseldenis Sorry for the delay. I was a bit silly when I did this PR. I did the format patch from 14.0 to 15.0 and then cherry pick the changes from @marielejeune instead of format patching from 16.0 to 15.0

@njeudy It was exactly that. 

Thanks for the reviews :)

**Review by rousseldenis — APPROVED** [review]

**Comment by NL66278** [comment]
@jdidderen-noviat Please squash some of the translation commits (for instance the consecutive ones for Croatian or Italian).

**Comment by ghost** [comment]
@NL66278 Done

File: partner_identification/__manifest__.py [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+#             http://www.camptocamp.at
+# Copyright 2015 Antiun Ingenieria, SL (Madrid, Spain)
+#        http://www.antiun.com
+#        Antonio Espinosa <antonioea@antiun.com>
+# Copyright 2016 ACSONE SA/NV (<http://acsone.eu>)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+{
+    "name": "Partner Identification Numbers",
+    "category": "Customer Relationship Management",
+    "version": "15.0.1.0.1",
```
Comment (MiquelRForgeFlow) [inline partner_identification/__manifest__.py]:
```suggestion
    "version": "15.0.1.0.0",
```

**Review by LoisRForgeFlow — APPROVED** [review]

File: partner_identification/__manifest__.py (hunk 1 above)
Comment (ghost) [reply partner_identification/__manifest__.py]: @MiquelRForgeFlow The change is applied. Thanks for the review :)

**Review by MiquelRForgeFlow — APPROVED** [review]

**Pull request merged**